Index: policorp/src/nautilus-pathbar.c
===================================================================
--- policorp.orig/src/nautilus-pathbar.c
+++ policorp/src/nautilus-pathbar.c
@@ -284,7 +284,7 @@ nautilus_path_bar_init (NautilusPathBar
     GtkEventController *controller;
     g_autoptr (GError) error = NULL;
 
-    self->os_name = g_get_os_info (G_OS_INFO_KEY_NAME);
+    self->os_name = g_get_os_info (G_OS_INFO_KEY_PRETTY_NAME);
 
     self->scrolled = gtk_scrolled_window_new ();
     /* Scroll horizontally only and don't use internal scrollbar. */
Index: policorp/src/nautilus-properties-window.c
===================================================================
--- policorp.orig/src/nautilus-properties-window.c
+++ policorp/src/nautilus-properties-window.c
@@ -849,7 +849,7 @@ update_name_field (NautilusPropertiesWin
 
     if (!is_multi_file_window (self) && is_root_directory (get_file (self)))
     {
-        os_name = g_get_os_info (G_OS_INFO_KEY_NAME);
+        os_name = g_get_os_info (G_OS_INFO_KEY_PRETTY_NAME);
         name_value = (os_name != NULL) ? os_name : _("Operating System");
     }
     else
